Source/AliveLib/DDCheat.cpp

```diff
@@ -131,7 +135,45 @@ void DDCheat::Menu_Teleport_415E20()
 
 void DDCheat::Menu_Movies_416000()
 {
-    NOT_IMPLEMENTED();
+    if (field_38_input_pressed & InputCommands::eLeft)
+    {
+        sDDCheat_MovieSelectIdx_5BBFF0--;
+    }
+    else if (field_38_input_pressed & InputCommands::eRight)
+    {
+        sDDCheat_MovieSelectIdx_5BBFF0++;
+    }
+
+    if (Path_Get_FMV_Record_460F70(gMap_5C3030.sCurrentLevelId_5C3030, sDDCheat_MovieSelectIdx_5BBFF0)->field_4_id <= 0 )
```

Comment:
you know it might bet better to put Path_Get_FMV_Record_460F70(gMap_5C3030.sCurrentLevelId_5C3030, sDDCheat_MovieSelectIdx_5BBFF0) into a local because its looked up 3 times in here

Reply:
I disagree, for two reasons:
-the first call might return a different value than the other two, as the if block can modify sDDCheat_MovieSelectIdx_5BBFF0 
-the latter two calls are independent, and won't happen simultaneously (unless someone holds both up and down arrow, which I'd consider OG bug)